### PR TITLE
Cleanup error code

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0080.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 enum Enum {
     X = (1 << 500),
-    Y = (1 / 0)
+    Y = (1 / 0),
 }
 ```
 
+This error indicates that the compiler was unable to sensibly evaluate a
+constant expression that had to be evaluated. Attempting to divide by 0
+or causing integer overflow are two ways to induce this error.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0080.md]:
```suggestion
or causing an integer overflow are two ways to induce this error.
```

File: src/librustc_error_codes/error_codes/E0081.md [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
     P = 3,
-    X = 3,
+    X = 3, // error!
     Y = 5,
 }
 ```
 
+Enum discriminants are used to differentiate enum variants stored in memory.
+This error indicates that the same value was used for two or more variants,
+making them impossible to tell apart.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0081.md]:
```suggestion
making it impossible to distinguish them.
```

**Review by Dylan-DPC-zz — DISMISSED** [review]

**Comment by GuillaumeGomez** [comment]
Updated!

**Pull request merged**